src/components/common/ListMapSelector.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Box, Theme, Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import Icon from './icon/Icon';
+import { IconName } from './icon/icons';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  icon: {
+    '& path': {
+      fill: theme.palette.TwClrBaseGray500,
+    },
+  },
+  box: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+  },
+}));
+
+export type View = 'list' | 'map';
+
+export type ListMapSelectorProps = {
+  view: View;
+  onView: (view: View) => void;
+};
+
+export default function ListMapSelector({ view, onView }: ListMapSelectorProps): JSX.Element {
+  const classes = useStyles();
+  const { isMobile } = useDeviceInfo();
+  const theme = useTheme();
+
+  const renderSelector = (iconName: IconName, title: string, selectorView: View) => (
+    <Box
+      className={classes.box}
+      width={isMobile ? '32px' : '67px'}
```

Comment:
Wide enough to account for translations? In Spanish "map" and "list" are "mapa" and "lista," respectively (according to Google anyways). I wonder if '67px' should be a min-width?

Reply:
Thanks for flagging, will try that.

Reply:
with min-width and a few other adjustments... (yet to check-in)

<img width="503" alt="ListMapSelector - Default ⋅ Storybook 2023-05-31 11-09-08" src="https://github.com/terraware/terraware-web/assets/1865174/fabcc53b-1c2c-48fe-9fc7-222da22b7fd1">

<img width="356" alt="ListMapSelector - Default ⋅ Storybook 2023-05-31 11-08-49" src="https://github.com/terraware/terraware-web/assets/1865174/75fe1057-c8bd-4f48-b856-781b23b3637f">